fastlane/Fastfile

```diff
@@ -770,6 +776,51 @@ end
       UI.user_error!("This lane should be run only from CI")
     end
   end
+
+  desc 'Registers a Device in the developer console'
+  lane :register_new_device do |options|
```

Comment:
The following logic has been tried and tested dozens of times across our other apps.